AUV/auv_viewer_processing/reconvert_thumb.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# before usage, please generate list of tiff files
+# \o auv_images_ls
+# select 'IMOS/AUV/' || campaign_code || '/' || site_code || '/' || image_folder
+# || '/' || image_filename || '.tif' from auv_viewer_track.auv_images_vw;
+
+
+import gzip
+import os
+import re
+
+from urllib2 import urlopen
+from functools import partial
+from wand.image import Image
+from multiprocessing import Pool, cpu_count
+
+
+def list_files(fname):
+    "read content of text file into list. handle tgz file (smaller for github)"
+    if fname.endswith('.gz'):
+        with gzip.open(fname, 'rb') as f:
+            return f.readlines()
+
+    else:
+        with open(fname) as f:
+            return f.readlines()
+
+
+def _generate_geotiff_thumbnail(thumbnail_dir_path, geotiff_name):
+    """
+    download the orginal file on S3
+    generate the thumbnail of one image
+    """
+    geotiff_path       = [s for s in geotiff_ls if geotiff_name in s][0]
+    s3_bucket_url      = 'http://s3-ap-southeast-2.amazonaws.com/imos-data/'
+    thumbnail_rel_path = re.sub('i2.*gtif', 'i2jpg', geotiff_path, flags=re.DOTALL).replace('IMOS/AUV/', 'IMOS/AUV/auv_viewer_data/thumbnails/').replace('.tif', '.jpg')
```

Comment:
I don't understand why you use `flags=re.DOTALL`. Could there be newlines in `geotiff_path`?